usage.md

```diff
@@ -1,175 +1,181 @@
 # Arcana Storage Usage Guide
 
+**Contents**
+
+1. [Prerequisites](#prerequisites)
+2. [Usage Flow](#usage-flow)
+3. [Initialize Storage](#initialize-storage)
+4. [Access Control](#access-control)
+5. [Storage Usage Metrics](#storage-usage-metrics)
+6. [Download File by DID](#download-file-by-did)
+7. [Private NFT](#private-nft)
+8. [Managing Network and Wallet Account Changes](#managing-network-and-wallet-account-changes)
+
 ## Prerequisites
 
-To use the Storage SDK, you need to ensure that you have access to a blockchain `provider`.  This is required for signing blockchain transactions that power data access control in the Arcana protocol.
+Arcana Storage SDK provides data privacy and access control through blockchain transactions powered by the Arcana Network. To use the Storage SDK, you need to ensure that you have access to a Web3 wallet `provider`.  A provider is required for signing blockchain transactions.
```

Comment:
Provider here is not referencing a piece of code. Code blocks are not appropriate.

Reply:
removed code block.